lib/events/user.ts

```diff
@@ -11,25 +13,35 @@ export default class UserEvent extends EventEmitter {
     this.crowi = crowi
   }
 
-  async onActivated(user) {
+  async onActivated(user: UserDocument) {
     const Page = this.crowi.model('Page')
-
     const userPagePath = Page.getUserPagePath(user)
-    Page.findPage(userPagePath, user, {}, false)
-      .then(function (page) {
-        // do nothing because user page is already exists.
-      })
-      .catch(function (err) {
-        const body = `# ${user.username}\nThis is ${user.username}'s page`
-        // create user page
-        Page.createPage(userPagePath, body, user, {})
-          .then(function (page) {
-            // page created
-            debug('User page created', page)
-          })
-          .catch(function (err) {
-            debug('Failed to create user page', err)
-          })
-      })
+
+    try {
+      const page = await Page.findPage(userPagePath, user, {}, false)
+
+      if (page === null) {
+        throw new Error()
+      }
+
+      // User page is already exists.
+      const renamedUserPagePath = `/tmp/user-${user.username}-${Date.now()}`
+      await Page.rename(page, renamedUserPagePath, user, {})
```

Comment:
💬 We have to decide how to handle pages under the user page manually created.

- Stay exist on the original path (Move pages to another place manually if user want to move pages under the user page that might created by the other user) 
- Or ... also move all pages under the user page at the same time.

Reply:
My opinion is "stay exist on the original path".

When moving all pages under the user page, links to moved pages in documents are broken.
I think the page should be changed at the user's discretion.

Reply:
@lightnet328 @sotarok Which is your opinion?

Reply:
> My opinion is "stay exist on the original path".

Agree with you!